### ISSUE TITLE
Validation failures for form.select

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1175]
Seems to be caused by the use of `aria-required`. Removing that attribute and relying on `required` fixes the error. Investigate if `aria-required` is still needed alongside `required` for all input types.

![image](https://user-images.githubusercontent.com/756393/77058836-e9cd9d00-69cd-11ea-9c87-d3e64c77f4a7.png)

**Comment by jamesjacobs** [comment]
Fixed by #1191
